build/gen_stub.php

```diff
@@ -1813,7 +1813,7 @@ private function getParameterSection(DOMDocument $doc): DOMElement {
             $parametersRefSec->appendChild($noParamEntity);
             return $parametersRefSec;
         } else {
-            $parametersPara = $doc->createElement('simpara');
+            $parametersPara = $doc->createElement('para');
             $parametersRefSec->appendChild($parametersPara);
```

Comment:
It shouldn't generate a `<para>` at all actually

Reply:
Then I guess it should be a document fragment, I can make that change shortly